src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -2629,7 +2638,7 @@ public async Task<NodeConnectionTestResult> TestConnectionToRemote(string url, s
 
                 using (var cts = new CancellationTokenSource(Server.Configuration.Cluster.OperationTimeout.AsTimeSpan))
                 {
-                    connectionInfo = ReplicationUtils.GetTcpInfoAsync(url, database, "Test-Connection", Server.Certificate.Certificate,
+                    connectionInfo = ReplicationUtils.GetTcpInfoAsync(url, database, null, default, "Test-Connection", Server.Certificate.Certificate,
```

Comment:
Create an overload, instead of add these everywhere

Reply:
done